conventions/src/main/kotlin/otel.sdk-extension.gradle.kts

```diff
@@ -0,0 +1,14 @@
+// SDK extensions are very similar to library instrumentations, they can be used without the javaagent
+// but since they depend on the SDK they must be loaded by the agent CL in the javaagent
+
+plugins {
+  id("io.opentelemetry.instrumentation.library-instrumentation")
+
+  id("otel.jacoco-conventions")
```

Comment:
Since we don't publish the jacoco reports maybe we shouldn't use it at all?

Reply:
Hm, I suppose that'd make sense.
I think there's a broader question of what we want to do with jacoco in general, cause it's currently being included in several gradle plugins (including the library instrumentation one), and we're not doing anything with that.